src/System.Private.CoreLib/shared/System/String.Manipulation.cs

```diff
@@ -1686,6 +1686,20 @@ public string Substring(int startIndex, int length)
             return InternalSubString(startIndex, length);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public string Substring(Index startIndex)
+        {
+            int actualIndex = startIndex.GetOffset(Length);
+            return Substring(actualIndex);
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public string Substring(Range range)
+        {
+            var (start, length) = range.GetOffsetAndLength(Length);
```

Comment:
For all of these uses of var, can we instead make them:
```C#
(int offset, int length) = …;
```
?

Reply:
I'll do that